examples/migrations_example/test/drift/default/migration_test.dart

```diff
@@ -0,0 +1,204 @@
+  "default - migrate from v1 to v2",
+  () => testStepByStepigrations(
+    from: 1, to: 2, verifier: verifier, createOld: (e) => v1.DatabaseAtV1(e),
+    createNew: (e) => v2.DatabaseAtV2(e), openTestedDatabase: (e) => Database(e),
+    createItems: (b, oldDb) {
+      b.insertAll(oldDb.users, v1_to_v2.usersV1);
+    },
+    validateItems: (newDb) async {
+      expect(v1_to_v2.usersV2, await newDb.select(newDb.users).get());
+    },
+  )
+);
+
+test(
+  "default - migrate from v2 to v3",
+  () => testStepByStepigrations(
+    from: 2, to: 3, verifier: verifier, createOld: (e) => v2.DatabaseAtV2(e),
+    createNew: (e) => v3.DatabaseAtV3(e), openTestedDatabase: (e) => Database(e),
+    createItems: (b, oldDb) {
+      b.insertAll(oldDb.users, v2_to_v3.usersV2);
+    },
+    validateItems: (newDb) async {
+      expect(v2_to_v3.usersV3, await newDb.select(newDb.users).get());
+    },
+  )
+);
+
+test(
+  "default - migrate from v3 to v4",
+  () => testStepByStepigrations(
+    from: 3, to: 4, verifier: verifier, createOld: (e) => v3.DatabaseAtV3(e),
+    createNew: (e) => v4.DatabaseAtV4(e), openTestedDatabase: (e) => Database(e),
+    createItems: (b, oldDb) {
+      b.insertAll(oldDb.users, v3_to_v4.usersV3);
+b.insertAll(oldDb.groups, v3_to_v4.groupsV3);
+    },
+    validateItems: (newDb) async {
+      expect(v3_to_v4.usersV4, await newDb.select(newDb.users).get());
+expect(v3_to_v4.groupsV4, await newDb.select(newDb.groups).get());
+    },
+  )
+);
+
+test(
+  "default - migrate from v4 to v5",
+  () => testStepByStepigrations(
+    from: 4, to: 5, verifier: verifier, createOld: (e) => v4.DatabaseAtV4(e),
+    createNew: (e) => v5.DatabaseAtV5(e), openTestedDatabase: (e) => Database(e),
+    createItems: (b, oldDb) {
+      b.insertAll(oldDb.users, v4_to_v5.usersV4);
+b.insertAll(oldDb.groups, v4_to_v5.groupsV4);
+    },
+    validateItems: (newDb) async {
+      expect(v4_to_v5.usersV5, await newDb.select(newDb.users).get());
+expect(v4_to_v5.groupsV5, await newDb.select(newDb.groups).get());
+    },
+  )
+);
+
+test(
+  "default - migrate from v5 to v6",
+  () => testStepByStepigrations(
+    from: 5, to: 6, verifier: verifier, createOld: (e) => v5.DatabaseAtV5(e),
+    createNew: (e) => v6.DatabaseAtV6(e), openTestedDatabase: (e) => Database(e),
+    createItems: (b, oldDb) {
+      b.insertAll(oldDb.users, v5_to_v6.usersV5);
+b.insertAll(oldDb.groups, v5_to_v6.groupsV5);
+    },
+    validateItems: (newDb) async {
+      expect(v5_to_v6.usersV6, await newDb.select(newDb.users).get());
+expect(v5_to_v6.groupsV6, await newDb.select(newDb.groups).get());
+    },
+  )
+);
+
+test(
+  "default - migrate from v6 to v7",
+  () => testStepByStepigrations(
+    from: 6, to: 7, verifier: verifier, createOld: (e) => v6.DatabaseAtV6(e),
+    createNew: (e) => v7.DatabaseAtV7(e), openTestedDatabase: (e) => Database(e),
+    createItems: (b, oldDb) {
+      b.insertAll(oldDb.users, v6_to_v7.usersV6);
+b.insertAll(oldDb.groups, v6_to_v7.groupsV6);
```

Comment:
Just noticed this - please run `DartFormatter` in `drift_dev` before emitting code, all generated files must be formatted.